.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib"
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
I want to propose the following lines to this file
```
  "editor.formatOnSave": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

It's a personal preference that allows Prettier to format the code when the file is saved. A proposal so not necessary!